### ISSUE TITLE
Question about writing 8 bytes in an install function like `Install_BigStash()`

### ISSUE THREAD

**Issue by pairofdocs** [fearedbliss/Alpaca#6]
Hi @fearedbliss 

I was looking at PlugY code, BH 1.9.9 code and I found your repo through [d2mods.info](https://d2mods.info/forum/viewtopic.php?t=63666)

for a function similar to `Install_BigStash()` ,
( https://github.com/fearedbliss/Alpaca/blob/bcd52d8eda8fad207773f712a1802f42ba3372a6/Source/Alpaca/BigStash.cpp#L104-L125 )

like this new function to install a FPS unlock feature for Single Player
```cpp
void Install_FPSUnlock()
{
	static bool isInstalled = false;
	if (isInstalled) return;

	Install_AlpacaFiles();

	log_msg("[Patch] Unlocked SP FPS Cap \n");

	DWORD FPSOffset = D2Client::GetAddress(0xHEX_HERE);

	// Modification
	Memory::SetCursor(FPSOffset);
        // Change 8 bytes here
```

What would be a good way to write 8 bytes of `0x90` NOPs ?

I see `void Memory::ChangeByte(BYTE oldValue, BYTE newValue)` and `void Memory::ChangeDword(DWORD oldValue, DWORD newValue)`

But those look like they write 1 byte or 4 bytes

Thank you

-------
References 

BH FPS feature offset (for 1.13c `0x44E51` and for 1.13d `0x45EA1`) and length 8 NOP write: https://github.com/planqi/slashdiablo-maphack/blob/abe525b4daa3e7377b37f9c264816b7a3ea8e35f/BH/Modules/Maphack/Maphack.cpp#L24

**Comment by fearedbliss** [comment]
Doesn't seem like this is related to an actual problem with Alpaca. Closing.

**Comment by fearedbliss** [comment]
@pairofdocs I replied to your thread on reddit, but for the record here, I've implemented a modified and safer FPS Unlock in Singling. The reason I didn't do it in Alpaca is because Alpaca is meant to be only the Infinite Stash component of PlugY in a more minimal, and stabilized way.

For any other modifications to the game such as game fixes (primarily fixes that don't change gameplay elements), those go exclusively in Singling. Singling and Alpaca work together as complements, providing similar functionality to vanilla d2, while not making any of the two projects deal with code outside of their primary responsibilities.

https://www.reddit.com/r/diablo2/comments/lvoe4s/cactus_fps_unlock_fix_now_available_for_both/

@pairofdocs Thanks overall for motivating me to look into this issue. For me, I've been playing for more than 15 years and still play almost daily, but the 25 FPS mouse issue doesn't really bother me. However, I have to admit, after implementing it, it does feel a lot better and smoother.
